app/views/govuk_publishing_components/components/layout_header/_navigation_items.html.erb

```diff
@@ -1,17 +1,24 @@
 <% if navigation_items.any? %>
-  <button role="button" class="govuk-header__menu-button gem-c-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
-  <%= tag.nav(class: "gem-c-header__nav", 'aria-label': navigation_aria_label ? navigation_aria_label : nil ) do %>
+  <div class="govuk-header__navigation-container">
+    <button role="button" class="govuk-header__menu-button gem-c-header__menu-button govuk-js-header-toggle" aria-controls="navigation" aria-label="Show or hide Top Level Navigation">Menu</button>
+    <%= tag.nav(class: "gem-c-header__nav", 'aria-label': navigation_aria_label ? navigation_aria_label : nil ) do %>
```

Comment:
Non-blocker, stylistic: aware this line wasn't introducted in this PR, but we don't need the rounded brackets here and I don't see the value in this short conditional statement `navigation_aria_label ? navigation_aria_label : nil`, I think we can contract it.
```suggestion
    <%= tag.nav class: "gem-c-header__nav", aria: { label: navigation_aria_label } do %>
```

Reply:
Thanks for the suggestion, it looks much cleaner.